allauth/account/adapter.py

```diff
@@ -676,6 +679,16 @@ def get_login_stages(self):
             ret.append("allauth.mfa.stages.AuthenticateStage")
         return ret
 
+    def send_notification_mail(self, template_prefix, user, context):
+        if app_settings.ACCOUNT_EMAIL_NOTIFICATIONS:
+            context |= {
+                "timestamp": datetime.now(),
```

Comment:
Can you use `django.utils.timezone.now()` here -- this way we will have a timestap with timezone.